sample-apps/everything-app/Dockerfile

```diff
@@ -1,17 +1,3 @@
-# OpenLiberty
 FROM openliberty/open-liberty:full-java8-openj9-ubi
 COPY src/main/liberty/config /config/
-ADD build/libs/myservice.war /config/dropins/
-
-# Wildfly
-#FROM jboss/wildfly
-#ADD build/libs/myservice.war /opt/jboss/wildfly/standalone/deployments/
-
-# Payara
-#FROM payara/micro:5.193
-#CMD ["--deploymentDir", "/opt/payara/deployments", "--noCluster"]
-#ADD build/libs/myservice.war /opt/payara/deployments
-
-# TomEE (not working yet)
-#FROM tomee:8-jre-8.0.0-M2-microprofile
-#COPY build/libs/myservice.war /usr/local/tomee/webapps/
```

Comment:
I would keep these comments.  They allow users of different application servers a starting point to test out MicroshedTesting with their applications.

Reply:
Each sample app has their own setup right? Why leave them in for all?

Reply:
As in having a separate Dockerfile for each application server?